packages/components/src/sandbox/test/index.js

```diff
@@ -14,62 +13,58 @@ import { useState } from '@wordpress/element';
  */
 import Sandbox from '../';
 
-let container;
+describe( 'Sandbox', () => {
+	global.window.MutationObserver = null;
```

Comment:
Why is this here? I'm surprised it's not breaking the code you've changed in the underlying implementation, considering we're expecting to call it as a constructor.

And just to be sure, this only affects this specific test file, not `MutationObserver` in other tests?

Reply:
Haha, I forgot to remove it but it doesn't have any impact on the code in this test suite 🤷‍♂